MVend/gwiza-dashboard
# ft():refactor folder structure

#### Description of changes made
---

- Change folder structure of the project

#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/MVend/gwiza-dashboard.git), and `cd [the project directory]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `yarn` to install the project dependencies
- Run `yarn dev` to start the server in the development mode.


#### Checklist:


#### Related issue

#

#### Blocker
> **If there is any blocker you have just mention it below**

N/A

#### Screenshots
N/A
